tests/python/test_000_runtime.py

```diff
@@ -0,0 +1,50 @@
+import taichi as ti
+
+
+# The first test to run, ever:
+def test_000_without_init():
```

Comment:
What does `000` mean here and in the file name?

Reply:
Seems `pytest` will starts from the lowest character order, so I use `000` to make `test_000_without_init` starts at the very first time when `ti.init()` has never been called in `@ti.all_archs`, to test if Taichi is functional even without `ti.init()`.

Reply:
Interesting - maybe we should simply let `ti.reset` clear everything, as if no `ti.init()` has been called? This will make the system more testable.

Reply:
> `ti.reset()`

I want to test if non of `ti.reset` or `ti.init` is called is working.
So, I'm creating a sandbox Taichi by creating a `taichi_temp_test.py` for test.

Btw, we may add a `ti.reset()` to taichi startup script to make your test method work? Not sure if this can crash doc-bots & make startup slower.